core/src/test/resources/spark-events/local-1555466577611

```diff
@@ -0,0 +1,15 @@
+{"Event":"SparkListenerLogStart","Spark Version":"3.0.0-SNAPSHOT"}
```

Comment:
Add local-1555466577611 to https://github.com/apache/spark/blob/master/dev/.rat-excludes

Reply:
Thanks @wangyum , I'll update